external/dir.targets

```diff
@@ -1,33 +1,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <ProjectJsonTemplate Condition="'$(ProjectJsonTemplate)' == ''">$(MSBuildThisProjectDirectory)project.json.template</ProjectJsonTemplate>
-    <ProjectJson Condition="Exists('$(ProjectJsonTemplate)')">$(IntermediateOutputPath)project.json</ProjectJson>
-    <ProjectLockJson Condition="Exists('$(ProjectJsonTemplate)')">$(IntermediateOutputPath)project.lock.json</ProjectLockJson>
-  </PropertyGroup>
-
   <Import Project="..\dir.targets" />
 
   <PropertyGroup>
     <RestorePackages>true</RestorePackages>
```

Comment:
I believe so... `RestorePackages` defaults to false and is only set in this folder.